docs/extend/modular-resources/key-concepts.md

```diff
@@ -59,7 +59,7 @@ Models are either:
 
 ### Built-in models
 
-Viam provides many built-in models that implement API capabilities, each using `rdk` as the `namespace`, and `builtin` as the `family`.
+Viam provides many built-in models that implement API capabilities, each using `rdk` as the `namespace`, and `builtin` as the `type`.
```

Comment:
"type" doesn't seem right here--`builtin` isn't component vs service, nor is it a subtype, so shouldn't this be a different word (maybe back to family? or repo-name? or just "designator" or something?)